src/proteomes/components/entry/Components.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, DataTable, ExternalLink } from 'franklin-sites';
+
+import useItemSelect from '../../../shared/hooks/useItemSelect';
+
+import ComponentsButtons from './ComponentsButtons';
+
+import externalUrls from '../../../shared/config/externalUrls';
+import { LocationToPath, Location } from '../../../app/config/urls';
+
+import {
+  ProteomesAPIModel,
+  Component,
+} from '../../adapters/proteomesConverter';
+
+import '../styles/overview.scss';
+
+const genomeAccessionDB = 'GenomeAccession' as const;
+
+export const Components: FC<Pick<ProteomesAPIModel, 'components' | 'id'>> = ({
+  components,
+  id,
+}) => {
+  const [selectedEntries, handleItemSelection] = useItemSelect();
+
+  if (!components?.length) {
+    return null;
+  }
+
+  const columns: Array<{
+    label: ReactNode;
+    name: string;
+    render: (arg: Component) => ReactNode;
+    width?: string;
+    ellipsis?: boolean;
+  }> = [
+    {
+      label: 'Component name',
+      name: 'component_name',
+      render: ({ name }) => name,
+    },
+    {
+      label: 'Genome accession(s)',
+      name: 'genome_accession',
+      render: ({ proteomeCrossReferences }) =>
+        proteomeCrossReferences
+          .filter(({ database, id }) => id && database === genomeAccessionDB)
+          .map<React.ReactNode>(({ id }) => (
+            <ExternalLink url={externalUrls.ENABrowser(id as string)} key={id}>
+              {id}
+            </ExternalLink>
+          ))
+          .reduce(
+            (prev, curr) => (prev === null ? [curr] : [prev, ', ', curr]),
```

Comment:
Maybe we should avoid using reduce for this?
A Fragment in the map starting with a `, ` for all but index 0 should be enough

Reply:
Done

Reply:
👍🏽
Not sure if it's wanted but now you don't have the extra space after the comma anymore

Reply:
Good catch - I did want that extra space so it's not crowded